Makefile.am

```diff
@@ -84,6 +82,795 @@ pkginclude_HEADERS += include/tesseract/strngs.h
 pkginclude_HEADERS += include/tesseract/thresholder.h
 pkginclude_HEADERS += include/tesseract/unichar.h
 
+# Rules for all subdirectories.
+
+noinst_HEADERS =
+noinst_LTLIBRARIES =
+
+AM_CPPFLAGS += -DLOCALEDIR=\"$(localedir)\"
```

Comment:
This flag has no effect.

https://github.com/tesseract-ocr/tesseract/search?q=LOCALEDIR&unscoped_q=LOCALEDIR

Reply:
You are right, thanks. I removed that old code.